assets/app/view/game/dividend.rb

```diff
@@ -161,6 +161,42 @@ def create_dividend(input)
         amount = input.JS['elm'].JS['value'].to_i * @step.current_entity.total_shares
         process_action(Engine::Action::Dividend.new(@step.current_entity, kind: 'variable', amount: amount))
       end
+
+      def dividend_chart
+        header, *chart = @step.chart
```

Comment:
isn't this going to break every game that calls this method that doesn't implement it

Reply:
There is exactly one game that uses variable dividend type: 18Mag

Reply:
My thinking is that a chart should always be displayed with a variable dividend since it can't be precomputed. I can test before calling, though.